test/integration.test.js

```diff
@@ -74,6 +74,27 @@ describe('Page', () => {
       }
     });
 
+    describe('Feeds', () => {
+      it(`fetch an XML feed`, (done) => {
+        sc.socket.emit('message', {
+          context: "feeds",
+          type: "fetch",
+          actor: {
+            type: "website",
+            id: "https://sockethub.org/examples/feeds"
+          },
+          target: {
+            type: 'feed',
+            id: 'http://localhost:10550/examples/feed.xml'
+          }
+        }, (msg) => {
+          console.log(`Feed fetch callback! `, msg);
+          expect(msg.length).to.eql(20);
+          done();
+        });
+      });
+    })
+
     describe('XMPP', () => {
       it('sends credentials', (done) => {
```

Comment:
This is missing a describe block for what exactly is expected to send credentials.